hackIT/templates/challenges.html.tera

```diff
@@ -3,8 +3,10 @@
   <head></head>
```

Comment:
No css for poor `challenges.html.tera`?

Reply:
i completely whiffed on this one! it should have some css and with an attempt of making it look nicer (still needs work though)